shared/src/summarize_result.js

```diff
@@ -54,7 +54,10 @@ async function setup() {
 
       if (data.success) {
         summaryResultElement.classList.remove('error');
-        summaryTextContents = data.summary;
+        summaryTextContents = new DOMParser().parseFromString(
+          data.summary.replaceAll(/<br>/g, '\n'),
+          'text/html',
+        ).documentElement.textContent;
```

Comment:
This was necessary because of some escaped HTML entities we receive from the response.